doc/usage.rst

```diff
@@ -173,6 +173,16 @@ The fmt port in vcpkg is kept up to date by Microsoft team members and community
 contributors. If the version is out of date, please `create an issue or pull
 request <https://github.com/Microsoft/vcpkg>`__ on the vcpkg repository.
 
+LHelper
+=======
+
+You can download and install fmt using `lhelper <https://github.com/franko/lhelper>`__ dependency manager::
+
+  lhelper activate <some-environment>
+  lhelper install fmt
+
+All the recipes for lhelper are kept in the `lhelper's recipe <https://github.com/franko/lhelper-recipes>`__ repository.
```

Comment:
Please break long lines for readability and consistency with the rest of the docs which use the limit of 80 when possible.